src/templates/project.tsx

```diff
@@ -322,6 +322,9 @@ export const query = graphql`
               # Highlight
 
               #text
+
+              # Embed
+              code
```

Comment:
Gatsby's GraphQL creates schemas automatically based on the queries.
Unfortunately, if we query `code` here, we need to make sure that at least one project .yml has this key.
Add it to the sample project so we don't have any problems with GraphQL